services/manifest-repo-export-service/pkg/cmd/server.go

```diff
@@ -141,7 +141,26 @@ func Run(ctx context.Context) error {
 			return fmt.Errorf("error converting KUBERPULT_ESL_PROCESSING_BACKOFF, error: %w", err)
 		}
 	}
-
+	var networkTimeoutSeconds uint64
+	if val, exists := os.LookupEnv("KUBERPULT_NETWORK_TIMEOUT_SECONDS"); !exists {
+		log.Infof("environment variable KUBERPULT_NETWORK_TIMEOUT_SECONDS is not set, using default timeout of 120 seconds")
+		networkTimeoutSeconds = 120
```

Comment:
We currently have 2 default values, 1 in the helm chart, one here.
We should only have 1 in the helm chart.
```suggestion
		return fmt.errorf...
```

Reply:
The helm chart is not the only way to start the service. It can also be started locally, either directly or with docker-compose, and I don't think this is an important environment variable that needs to be explicitly set for the service to start.

Reply:
As agreed, I set the variables to required